ArduCopter/Log.cpp

```diff
@@ -538,12 +485,6 @@ void Copter::Log_Write_Throw(ThrowModeStage stage, float velocity, float velocit
 // units and "Format characters" for field type information
 const struct LogStructure Copter::log_structure[] = {
     LOG_COMMON_STRUCTURES,
-#if AUTOTUNE_ENABLED == ENABLED
```

Comment:
I do not see where this block of code gets moved to !!!

Reply:
@amilcarlucas  i am a bit confused about where this block of code should be moved. Can you please guide me for that.